src/Symfony/Component/Console/Application.php

```diff
@@ -120,20 +120,28 @@ public function run(InputInterface $input = null, OutputInterface $output = null
 
         $this->configureIO($input, $output);
 
+        $exception = null;
+        $exitCode = null;
```

Comment:
the intend was to make code more readable and easy to understand.

but sure, this can be removed